nss-day-cohort-29/reactive-nutshell-copperden
# Ab events styling

# Description 
Updated styling to meet issue ticket needs.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout ab-events-styling`
3. Run server
4. Verify that nearest event is styled with a non-offensive background and bold title stating that it is the next event.

# Checklist:
